app/assets/stylesheets/pages/generic-page.scss

```diff
@@ -51,19 +51,22 @@
     p+ul {
         margin-top:-10px;
     }
-    p,ul,ol {
-        @include font-size(16, 23);
-        margin-bottom:15px;
-        strong {
-            font-weight:$bold;
-        }
-        a {
-            color:#333;
-            text-decoration:none;
-            border-bottom:1px solid #aaa;
-        }
-        li {
-            margin-bottom:5px;
+
+    .pure-g{
```

Comment:
Can we use another class here instead of `pure-g`?

Reply:
Would the other option be to add a new class?

Reply:
Yes. I think it shouldn't be a problem to add a new class.

Reply:
Okay, I added a new class.